help/actions/_posts/2015-02-09-export.md

```diff
@@ -140,10 +148,22 @@ The output field specifiers are added to the beginning of the custom format as d
 
     This is a special format directive. It must be the first command specified in a format string, and there can be multiple *%Z* directives at the beginning of the string. While the *Z* tag produces no output, it allows you to specify the value of certain variables with corresponding parameters. It has the following syntax: **%Zcommand:parameter**. The parameter can be a number, a word, or a string, depending on the command. If it is a string, it must be enclosed in double quotes. The available commands are: 
 
-    * **Encoding:word** This specifies the encoding of the output. Available encoding parameters are: 
-        * **unicode** (default): UTF-8 encoding of all unicode characters. 
+    There are two encoding options: global and field. As the name suggests specifying global encoding applies the encoding to the entire citiation, all output field. Please see further down for explanation of field encoding.
```

Comment:
* this line is a little confusing here; I'd move it down a line and integrate it with the text there, so it reads something like this (also there were a couple typos in the original text that I've fixed here):
**Encoding:word** Note that there are two encoding options: global and field. As the name suggests, specifying global encoding applies the encoding to the entire citation, or all output fields. Please see further down for an explanation of field encoding. Available global encoding parameters are: